src/desktop/apps/artwork/components/commercial/view.coffee

```diff
@@ -39,14 +40,18 @@ module.exports = class ArtworkCommercialView extends Backbone.View
   acquire: (e) ->
     e.preventDefault()
 
-    order = new PendingOrder
-    @form = new Form $form: @$('form'), model: order
+    loggedInUser = CurrentUser.orNull()
+    if loggedInUser && loggedInUser.hasLabFeature('New Buy Now Flow')
+      createOrder(loggedInUser, @artwork.get('partner_id'), "usd", @artwork.get('_id'), 500000, 1)
```

Comment:
How come `500000` is hardcoded?

Reply:
Ah, this is a TODO also. I don't think we currently pass price as a cents field in the artwork schema (or at all?) but I haven't looked much into it yet. Felt like a thing to follow up on.

Reply:
Sure 👍

Reply:
(that shd be currency as well possibly)